### PR TITLE
Indexed Data Cleaning Script

### PR DESCRIPTION
**Pull request by marcus-snx**
Adds a script (`clean.py`) in the indexer service that reorganizes Parquet files from the subsquid indexer raw output into a structure that matches the new taxonomy:

Input: `indexers/raw/{network}/{protocol}/{block_range}/{event}.parquet`
Output: `indexers/clean/{network}/{protocol}/{event}/{event}_{block_range}.parquet`

### CODE REVIEW

File: indexers/entrypoint.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 set -e
 
 # Get contract data from SDK and generate squidgen.yaml and squid.yaml
-python3 main.py --network_name $NETWORK_NAME --protocol_name $PROTOCOL_NAME "$@"
+uv run main.py --network_name $NETWORK_NAME --protocol_name $PROTOCOL_NAME "$@"
```
Comment (Tburm) [inline indexers/entrypoint.sh:7]: I noticed we use `network_name` and `protocol_name` here but `network` and `protocol` in the cleaning scripts. We should make it all match.

File: indexers/scripts/clean.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            df = pd.read_parquet(parquet_file)
+            event_dir.mkdir(parents=True, exist_ok=True)
+            df.to_parquet(output_file, index=False)
+        print(f"Processed {protocol} {block_range}")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--network", type=str, required=True)
+    parser.add_argument("--protocol", type=str, required=True)
```
Comment (Tburm) [inline indexers/scripts/clean.py:38]: see my comment about matching naming here for indexers, also I think this should come from the environment variables if they exist

**Comment by Tburm** [comment]
I also noticed you can't really gracefully restart a continuous indexer. I tried running it, stopping the container, then running it again. It looks like this:
```
data-indexer-1  | {"level":2,"time":1732062968268,"ns":"sqd:evm-typegen","msg":"processing ./abi/buyback_snx.json"}
data-indexer-1  | {"level":2,"time":1732062968285,"ns":"sqd:evm-typegen","msg":"saved src/abi/buyback_snx.abi.ts"}
data-indexer-1  | {"level":2,"time":1732062968286,"ns":"sqd:evm-typegen","msg":"saved src/abi/buyback_snx.ts"}
data-indexer-1  | {"level":2,"time":1732062968374,"ns":"sqd:squidgen","msg":"running codegen..."}
data-indexer-1  | {"level":2,"time":1732062968580,"ns":"sqd:squidgen","msg":"generating processor..."}
data-indexer-1  | mv: can't rename 'config.ts': No such file or directory
```

**Comment by marcus-snx** [comment]
> I also noticed you can't really gracefully restart a continuous indexer. I tried running it, stopping the container, then running it again. It looks like this:
> 
> ```
> data-indexer-1  | {"level":2,"time":1732062968268,"ns":"sqd:evm-typegen","msg":"processing ./abi/buyback_snx.json"}
> data-indexer-1  | {"level":2,"time":1732062968285,"ns":"sqd:evm-typegen","msg":"saved src/abi/buyback_snx.abi.ts"}
> data-indexer-1  | {"level":2,"time":1732062968286,"ns":"sqd:evm-typegen","msg":"saved src/abi/buyback_snx.ts"}
> data-indexer-1  | {"level":2,"time":1732062968374,"ns":"sqd:squidgen","msg":"running codegen..."}
> data-indexer-1  | {"level":2,"time":1732062968580,"ns":"sqd:squidgen","msg":"generating processor..."}
> data-indexer-1  | mv: can't rename 'config.ts': No such file or directory
> ```

Should be fixed now, I removed the `config.ts` file altogether as we were not using it anymore

**Review by Tburm — APPROVED** [review]

**Pull request merged**